api/src/main/java/org/apache/iceberg/NaNOnlyFieldMetrics.java

```diff
@@ -17,51 +17,50 @@
  * under the License.
  */
 
-package org.apache.iceberg.parquet;
```

Comment:
Does this need to be in API or could it be in `core` instead?

If the class is never returned to users, then I would keep it in core. The classes in `api` are primarily those that users would interact with.

Reply:
Sounds good, I'll move both this and `FieldMetrics` to core in both here and #1829